packages/twenty-front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx

```diff
@@ -166,7 +169,7 @@ export const SettingsObjectFieldEdit = () => {
               <SaveAndCancelButtons
                 isSaveDisabled={!canSave}
                 onCancel={() => navigate(`/settings/objects/${objectSlug}`)}
-                onSave={handleSave}
+                onSave={formConfig.handleSubmit(handleSave)}
```

Comment:
If I'm not mistaken, `handleSubmit` passes the validated formValues to `handleSave` as a parameter, so we shouldn't need to use `formConfig.getValues()` in `handleSave`?

Reply:
indeed, thanks!